### PR TITLE
fix(curriculum): external fibonacci word link in CIP

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Fibonacci Word #40749 

I have been looking at this piece of paper for 7 days and still can't wrap my head around it

### CODE REVIEW

**Review by Sembauke — DISMISSED** [review]

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ dashedName: fibonacci-word
...
 # --description--
 
-The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence [as described here](https://hal.archives-ouvertes.fr/docs/00/36/79/72/PDF/The_Fibonacci_word_fractal.pdf):
+The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence, but it focus on iterating concatenation, with terms different each time.
 
 <pre>Define  F_Word<sub>1</sub>  as  <strong>1</strong>
 Define  F_Word<sub>2</sub>  as  <strong>0</strong>
 Form   F_Word<sub>3</sub>  as  F_Word<sub>2</sub>   concatenated with  F_Word<sub>1</sub>   i.e.:  <strong>01</strong>
-Form   F_Word<sub>n</sub>  as  F_Word<sub>n-1</sub>  concatenated with  F_word<sub>n-2</sub>
+Form   F_Word<sub>term</sub>  as  F_Word<sub>term-1</sub>  concatenated with  F_word<sub>term-2</sub>
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:16]: I think the use of n is a bit more typical and consistent with the Fibonacci numbers themselves

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ dashedName: fibonacci-word
...
 </pre>
 
 # --instructions--
 
-Write a function to return the Fibonacci Words up to `n`. `n` will be provided as a parameter to the function. The function should return an array of objects. The objects should be of the form: `{ N: 1, Length: 1, Entropy: 0, Word: '1' }`. `Entropy` is computed for the string `Word` [as described here](https://en.wikipedia.org/wiki/Entropy_%28information_theory%29) and rounded to 8 decimal digits of accuracy.
+Write a function to return the Fibonacci Words up to `terms`. `terms` will be provided as a parameter to the function. The function should return an array of objects. The objects should be of the form: `{ term: 1, Length: 1, Entropy: 0, Word: '1' }`. `Entropy` is computed for the string `Word` and rounded to 8 decimal digits of accuracy.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:21]: Is this reasonably doable without providing one of the many definitions of Entropy for the learners to use?

Comment (Sboonny) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:21]: I will try to solve it tomorrow morning and update it accordingly.👍

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 3]
```diff
@@ -34,13 +34,13 @@ assert(typeof fibWord === 'function');
 assert(Array.isArray(fibWord(5)));
 ```
 
-`fibWord(5)` should return `[{ N:1, Length:1, Entropy:0, Word:"1" },{ N:2, Length:1, Entropy:0, Word:"0" },{ N:3, Length:2, Entropy:1, Word:"01" },{ N:4, Length:3, Entropy:0.91829583, Word:"010" },{ N:5, Length:5, Entropy:0.97095059, Word:"01001" }]`.
+`fibWord(5)` should return `[{ term:1, Length:1, Entropy:0, Word:"1" },{ term:2, Length:1, Entropy:0, Word:"0" },{ term:3, Length:2, Entropy:1, Word:"01" },{ term:4, Length:3, Entropy:0.91829583, Word:"010" },{ term:5, Length:5, Entropy:0.97095059, Word:"01001" }]`.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:37]: Changing
term -> Term 
would match the other fields in the output

**Comment by Sboonny** [comment]
@ Jeremy, do you think this is good enough for it to be solved internally.

> Entropy calculation is required in this challenge, [Here is a rosetta challenge for calculating entropy](https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/entropy)

**Comment by jeremylt** [comment]
As long as the formulae agree, totally good idea in my book

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 4]
```diff
@@ -52,28 +54,28 @@ assert.deepEqual(fibWord(7), words7);
...
+  { Term: 6, Length: 8, Entropy: 0.954434, Word: '01001010' },
+  { Term: 7, Length: 13, Entropy: 0.9612366, Word: '0100101001001' }
 ];
 ```
 
 ## --seed-contents--
 
 ```js
-function fibWord(n) {
+function fibWord(terms) {
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:78]: Text above says the argument will be `n`

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 5]
```diff
@@ -8,17 +8,19 @@ dashedName: fibonacci-word
 
 # --description--
 
-The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence [as described here](https://hal.archives-ouvertes.fr/docs/00/36/79/72/PDF/The_Fibonacci_word_fractal.pdf):
+The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence, but it focus on iterating concatenation, with terms different `n` each time.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:11]:
```suggestion
The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence, but it focus on iterating concatenation.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 6]
```diff
@@ -8,17 +8,19 @@ dashedName: fibonacci-word
...
 
 <pre>Define  F_Word<sub>1</sub>  as  <strong>1</strong>
 Define  F_Word<sub>2</sub>  as  <strong>0</strong>
 Form   F_Word<sub>3</sub>  as  F_Word<sub>2</sub>   concatenated with  F_Word<sub>1</sub>   i.e.:  <strong>01</strong>
-Form   F_Word<sub>n</sub>  as  F_Word<sub>n-1</sub>  concatenated with  F_word<sub>n-2</sub>
+Form   F_Word<sub>n</sub>  as  F_Word<sub>n-1</sub>  concatenated with  F_word<sub>Term-2</sub>
 </pre>
 
+Entropy calculation is required in this challenge, [Here is a rosetta challenge for calculating entropy](https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/entropy)
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:19]:
```suggestion
Entropy calculation is required in this challenge, [as shown in this Rosetta Code challenge](https://www.freecodecamp.org/learn/coding-interview-prep/rosetta-code/entropy)
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 7]
```diff
@@ -8,17 +8,19 @@ dashedName: fibonacci-word
...
+
 # --instructions--
 
-Write a function to return the Fibonacci Words up to `n`. `n` will be provided as a parameter to the function. The function should return an array of objects. The objects should be of the form: `{ N: 1, Length: 1, Entropy: 0, Word: '1' }`. `Entropy` is computed for the string `Word` [as described here](https://en.wikipedia.org/wiki/Entropy_%28information_theory%29) and rounded to 8 decimal digits of accuracy.
+Write a function to return the Fibonacci Words up to `terms`. `terms` will be provided as a parameter `n` to the function. The function should return an array of objects. The objects should be of the form: `{ Term: 1, Length: 1, Entropy: 0, Word: '1' }`. `Entropy` is computed for the string `Word` and rounded to 8 decimal digits of accuracy.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:23]:
```suggestion
Write a function to return the first `n` Fibonacci Words. The number of terms is provided as a parameter to the function. The function should return an array of objects. The objects should be of the form: `{ Term: 1, Length: 1, Entropy: 0, Word: '1' }`. `Entropy` is computed for the string `Word` and rounded to 8 decimal digits of accuracy. Note that the indices of this sequence start at `1`.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md [hunk 8]
```diff
@@ -8,17 +8,19 @@ dashedName: fibonacci-word
 
 # --description--
 
-The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence [as described here](https://hal.archives-ouvertes.fr/docs/00/36/79/72/PDF/The_Fibonacci_word_fractal.pdf):
+The Fibonacci Word may be created in a manner analogous to the Fibonacci Sequence, but it focus on iterating concatenation.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/fibonacci-word.md:11]: woops, one last one
```suggestion
The Fibonacci Word Sequence may be created in a manner analogous to the Fibonacci Sequence, but it focuses on iterating concatenation.
```

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Want to take one more look at this and merge if your satisfied @jeremylt?

**Review by jeremylt — APPROVED** [review]

**Pull request merged**